b2b-sample/guardio-insurance-sample-application/util/util/apiUtil/getSentDataRequestOptions.js

```diff
@@ -20,9 +20,10 @@ import config from '../../../config.json';
 
 function sentDataHeader(session) {
     const headers = {
-        "accept": "application/scim+json",
-        "content-type": "application/scim+json",
+        "accept": "application/json",
+        "content-type": "application/json",
         "authorization": "Bearer " + session.accessToken,
+        
```

Comment:
Do we need this new line?

Reply:
Fixed with the PR which moved the functionality to a different file,

### Commit
- https://github.com/Achintha444/samples-is/commit/84e5af312596e01473d17597ddf5641645503597#diff-968532fdc2f70738a9cbab5b80b9bd297eb489f158eea4fb3dce0db7062b4545